src/Alarm.cpp

```diff
@@ -24,7 +24,6 @@ void Alarm::calibrate(){
     alertSuccessfulAction();
     _isCalibrated = true;
   }
-  Serial.println("Alarm is calibrating");
```

Comment:
We need to address the removal of these debug statements. My vote would be to put them all back in but wrapped with the if check on the debugging flag in Properties.cpp.

If you want to do it then I'll wait. Otherwise I could work on putting them back in with that check during lunch this afternoon. Would you like me to commit directly to this branch and this Pull Request? Or would you prefer I branch off this branch and do a Pull Request back into it?